api/v1alpha1/nifidataflow_types.go

```diff
@@ -22,7 +22,6 @@ type NifiDataflowSpec struct {
 	// contains the reference to the ParameterContext with the one the dataflow is linked.
 	ParameterContextRef *ParameterContextReference `json:"parameterContextRef,omitempty"`
 	// if the flow will be synchronized once, continuously or never
-	// +kubebuilder:validation:Enum={"never","always","once"}
```

Comment:
Why this change ?

Reply:
To make this kubebuilder annotation available to all the resources using the DataflowSyncMode type to avoid having to copy and paste.